src/OpenDiffix.Core/Anonymizer.fs

```diff
@@ -247,6 +247,10 @@ let isLowCount (executionContext: ExecutionContext) (aidSets: HashSet<AidHash> s
   )
   |> Seq.reduce (||)
 
+type Result =
```

Comment:
I have a problem with naming here, suggestions welcome. 

In general my aim here is for the types to better represent the meaning. `Value` isn't good, because it can (completely nonsensically here) be a `String`.

A "last resort" alternative is `Option<int>`, but it is less meaningful, why the `None` in here is better described by the verbose `NotEnoughAIDVs`. I'd like this to be somehow captured, and set apart from the other "optionalities" involved around the code. `NotEnoughAIDVs` communicates to the `Aggregator` what needs to be done.

Reply:
How about `CountResult`? `Result` is a default type and I wouldn't prefer creating ambiguity.

Reply:
could work, but then how to rename the `Count` inside? or is `CountResult.Count` OK?

Reply:
Not sure, and also not important. Maybe: `CountResult.CountResult`, or `CountResult.Ok` and require explicit qualification for the union (needs an attribute).

Reply:
`CountResult.Ok` is nice